multiapps-controller-process/src/main/java/org/cloudfoundry/multiapps/controller/process/util/TimeoutType.java

```diff
@@ -0,0 +1,31 @@
+package org.cloudfoundry.multiapps.controller.process.util;
+
+import org.cloudfoundry.multiapps.controller.core.model.SupportedParameters;
+
+public enum TimeoutType {
+    UPLOAD, STAGE, START, TASK;
+
+    public static String getTimeoutModuleLevelParameterName(TimeoutType timeoutType) {
```

Comment:
No need for these methods - you can have private variables in the enum instances (e.g. String moduleLevelParamName and String globalLevelParamName) and then just get the values by calling timeoutType.getModuleLevelParamName();